### ISSUE TITLE
Page component - remove `<aside>` from nav section

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1185]
The `<aside>` element should be changed to a generic `<div>`.

Both `<aside>` and `<nav>` are [landmark regions](https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html) that serve different purposes. The `<nav>` in the page component should not be wrapped in an `<aside>`, but instead should just be a generic `<div>` element.

**Comment by mcoker** [comment]
Looks like this was addressed in https://github.com/patternfly/patternfly-next/pull/1188/
